src/aleph/sdk/client/abstract.py

```diff
@@ -395,10 +395,13 @@ async def create_instance(
         internet: bool = True,
         aleph_api: bool = True,
         hypervisor: Optional[HypervisorType] = None,
+        confidential_firmware: Optional[ItemHash] = None,
```

Comment:
Why not call this simply `firmware` ?
```suggestion
        firmware: Optional[ItemHash] = None,
```

Reply:
Because it's the firmware specific for the confidential instances.